docs/EmailBySendgrid-postaction.md

```diff
@@ -0,0 +1,49 @@
+# Sendgrid Post-run Action
+
+
+Description
+-----------
+The Sendgrid post-action can be used to send an email at the end of a pipeline run. You can configure it to send an 
+email when the pipeline completes, succeeds or fails.
```

Comment:
Should we add any information about what is needed in order to use this? A Sendgrid account? link to any doc?